### PR TITLE
Added LastMenuUpdateTime to menu template variables, improved relativeTime to handle timestamps in the past

### PR DESCRIPTION
**Pull request by seejdev**
This PR adds a `LastMenuUpdateTime` to the menu template variables. This enables control server to provide a debug menu item with custom text, and formats the last time the menu was updated as a fuzzy relative time.

Menu JSON:

```
{"label":"Last Menu Update: {{if hasCapability `relativeTime`}}{{relativeTime .LastMenuUpdateTime}}{{else}}{{end}}","tooltip":null,"disabled":true,"items":[]}
```

Expands to:

```
{"label":"Last Menu Update: 10 Seconds Ago","tooltip":null,"disabled":true,"items":[]}
```

and we get:
Screenshot 2023-04-17 at 1 34 43 PM

### CODE REVIEW

**Review by James-Pickett — DISMISSED** [review]

**Review by RebeccaMahany — DISMISSED** [review]

**Review by James-Pickett — DISMISSED** [review]

File: ee/desktop/menu/menu_template.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ type TemplateData struct {
 	LauncherRevision string `json:",omitempty"`
 	GoVersion        string `json:",omitempty"`
 	ServerHostname   string `json:",omitempty"`
+	LastUpdateTime   int64  `json:",omitempty"`
```
Comment (directionless) [inline ee/desktop/menu/menu_template.go:20]: I think I want to be clear about what this means. 

It's not the time k2 generates the data. It's it's the time we last noticed new data and wrote it to disk?

Comment (seejdev) [reply ee/desktop/menu/menu_template.go:20]: Correct, including if someone were to open the menu template file locally and modify it.

Comment (directionless) [reply ee/desktop/menu/menu_template.go:20]: Hrm.... If K2 starts including the time it was generated k2 side, does that change how we want to name things?
`LastServerGeneratedTime` and `LastUpdateTime`? Something else?

Comment (seejdev) [reply ee/desktop/menu/menu_template.go:20]: `LastServerGeneratedTime` and `LastMenuUpdateTime`?

Comment (directionless) [reply ee/desktop/menu/menu_template.go:20]: Sure

Comment (seejdev) [reply ee/desktop/menu/menu_template.go:20]: Actually, `LastServerGeneratedTime` would just come directtly from k2 - so no need for a launcher template variable.

Comment (directionless) [reply ee/desktop/menu/menu_template.go:20]: Agreed. I'm thinking about the names and what to call things.

Comment (seejdev) [reply ee/desktop/menu/menu_template.go]: Renamed `LastUpdateTime` to `LastMenuUpdateTime`

**Review by James-Pickett — APPROVED** [review]

**Pull request merged**